modules/ROOT/pages/csae.adoc

```diff
@@ -0,0 +1,88 @@
+= Getting started with Teradata VantageCloud ClearScape Analytics^TM^
```

Comment:
ClearScape Analytics is its own name. We don't use it with `Teradata VantageCloud`. Also, I'm worried that it might be confusing for people if we highlight ClearScape Analytics here. I think this should be focused on ClearScape Analytics Experience. The only time we mention ClearScape Analytics is when we list what can be explored in CSAE.

```suggestion
= Getting started with ClearScape Analytics Experience
```